vttools/wrap_lib.py

```diff
@@ -668,7 +668,7 @@ def compute(self):
         ret = library_func(**params_dict)
         if len(output_ports) == 1:
             self.set_output(output_ports[0].name, ret)
-        else:
+        elif ret:
```

Comment:
I am worried about the case where `len(output_ports) != len(ret)` so we will have output ports which exist from vistrails point of view, but never get a value set so the next node down the graph will break.

By scraper I mean the code that read the docstrings and tried to figure out all the input and output ports.